app/src/main/java/org/mozilla/fenix/tabstray/browser/UseCases.kt

```diff
@@ -13,19 +13,27 @@ class SelectTabUseCaseWrapper(
     private val selectTab: TabsUseCases.SelectTabUseCase,
     private val onSelect: (String) -> Unit
 ) : TabsUseCases.SelectTabUseCase {
-    override fun invoke(tabId: String) {
-        metrics.track(Event.OpenedExistingTab)
+    operator fun invoke(tabId: String, source: String? = null) {
```

Comment:
Didn't want to modify the AC interface to support the new `source` parameter since this is only used by our telemetry so I added an overloaded `invoke`().